soapui/build.gradle

```diff
@@ -2,12 +2,22 @@ dependencies {
     compileOnly 'com.smartbear.soapui:soapui:5.4.0'
 
     compileOnly 'log4j:log4j:1.2.17'
-    compile 'com.epam.reportportal:client-java:4.0.4'
-    compile 'com.epam.reportportal:logger-java-log4j:4.0.1'
+    compile 'com.github.reportportal:client-java:591b7b41'
+    compile 'com.github.reportportal:commons-model:feec39f3'
+    compile 'com.epam.reportportal:logger-java-log4j:5.0.0-BETA-8'
 }
 
 shadowJar {
     dependencies {
         exclude(dependency('com.smartbear.soapui:soapui'))
     }
-}
\ No newline at end of file
+    archiveName = "agent-java-${archiveName}"
+}
+
+task writeAgentProperties(type: WriteProperties) {
+    outputFile = file("${project.projectDir}/src/main/resources/agent.properties")
```

Comment:
As per @ihar-kahadouski :https://github.com/reportportal/agent-java-jbehave/pull/35#discussion_r376355434

```
    {
    filesMatching('agent.properties') {
        expand(project.properties)
    }
```
and in properties file:

```
agent.name=${name}
agent.version=${version}
```

Reply:
updated